### PR TITLE
git-brv: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

**Comment by bl-ue** [comment]
@CleanMachine1 because you have write access to the repository, you can (1) label your PRs, and (2) link your PR to the appropriate let's document issue.

File: pages/common/git-brv.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# git brv
+
+> Print a list of branches, sorted by their date since last commit.
```
Comment (bl-ue) [inline pages/common/git-brv.md:3]: More succinct?
```suggestion
> Print a list of branches, sorted by last commit date.
```

File: pages/common/git-brv.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-brv>.
+
+- Print a list of branches showing date, latest commit and latest commit hash for each branch:
```
Comment (bl-ue) [inline pages/common/git-brv.md:7]: "latest" could probably be omitted, wdyt?
```suggestion
- Print a list of branches showing date, latest commit and commit hash for each branch:
```

**Comment by CleanMachine1** [comment]
Yes, I forgot, doing it now

**Comment by CleanMachine1** [comment]
I don't want to link since it closes the #5137

**Comment by bl-ue** [comment]
It won't close it if your link it like this `For #...`.  GH only auto closes if it detects keywords such as `Fixes`, `Closes`, `Resolves`, etc.

**Comment by CleanMachine1** [comment]
done

File: pages/common/git-brv.md (hunk 2 above)
Comment (navarroaxel) [inline pages/common/git-brv.md:7]: Maybe we can make this shorter.
```suggestion
- List each branch showing date, latest commit and latest commit hash:
```

Comment (CleanMachine1) [reply pages/common/git-brv.md]: Yes, I will have to do it in a different commit since this is out of date

**Review by bl-ue — APPROVED** [review]
Lovely.  Also, big thanks → @Osse for developing this tool! (and @bric3 for popularizing it via `git-extras`!) ❤️ 🚀

File: pages/common/git-brv.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/common/git-brv.md]: Actually, I have added @navarroaxel edit

**Comment by bl-ue** [comment]
@CleanMachine1 I'd like to tell you: when you merge PRs, remember to delete the `Co-authored-by` lines, because GitHub adds them for anyone who submits a suggestion and gets that suggestion applied, and it can really hide the actually PR author when viewing the merge commit.

**Comment by CleanMachine1** [comment]
> @CleanMachine1 I'd like to tell you: when you merge PRs, remember to delete the `Co-authored-by` lines, because GitHub adds them for anyone who submits a suggestion and gets that suggestion applied, and it can really hide the actually PR author when viewing the final merge commit in the [tree view](https://github.com/tldr-pages/tldr/commits/main).

Does it matter?

**Comment by bl-ue** [comment]
It's really not _important_, but it looks a lot worse if you leave them in.  See commit https://github.com/tldr-pages/tldr/commit/e64ec920e7752bd5d8141a5cfc91834dc455a555 for an example.

**Comment by CleanMachine1** [comment]
If someone wants to view the users, they can hover over it and it singles them out, and the top person is the author

**Comment by bl-ue** [comment]
True, but then every almost every commit will have some of us in, because we have many things to say about most pages 😆

**Comment by CleanMachine1** [comment]
Commit count++, should be happy :smile:

**Comment by bl-ue** [comment]
But remember, the amount of coauthors doesn't increase the commit count.  10 commits each of which has 9 coauthors (in addition to the 1 author) isn't 100 commits - it's still 10 commits.

**Review by navarroaxel — APPROVED** [review]

**Review by bric3 — APPROVED** [review]

**Review by Waples — APPROVED** [review]

**Comment by navarroaxel** [comment]
@Waples please remember delete the commit message body when you `squash & merge`.

**Comment by Waples** [comment]
The android app strikes again, didn't get the option

**Pull request merged**